### ISSUE TITLE
Standings table, improved order

### ISSUE THREAD

**Issue by alex5749** [coolchess123/tcec-gui#227]
The ranking order of the Standings table is misleading when the programs haven't played the same number of games as each other (which is most of the time). For example (i) 1 draw + 1 loss might rank ahead of 1 draw, whereas it is clearly worse, and (ii) 1 draw + 1 win might rank below 1 draw, whereas it is clearly better.

Suggestion: rank by (#wins - #losses) / #gamesplayed
(taken to be 0 if #gamesplayed=0).

If number of games played is equal, then this is the same as ranking by points.

**Comment by coolchess123** [comment]
yes now it sorts and gives < games more weigtage

**Comment by alex5749** [comment]
That is a big improvement, thank you, but not quite the same as what I suggested. A program that has drawn 2 games and lost 100 will rank above one that has drawn 1 (because 1 > 0.5). I am suggesting that the 100 lost games should be taken into account using the formula I gave.

I've attached an example

![standings](https://user-images.githubusercontent.com/1798181/58506784-25218580-8188-11e9-8ccd-5525b33e6e1e.png)

where Glaurung is +1-4=1 and is ranked 3rd above all those on +0-0=0. I suggest it should be ranked last because it has lost more than it has won.
